### PR TITLE
CLEANUP: Remove xml module usage from bucketGetVersioning api

### PR DESCRIPTION
**Pull request by bennettbuchanan**
* Remove the dependency on xml module in bucketGetVersioning api
* Remove the xml module from package.json as it is no longer used
  in S3

### CODE REVIEW

File: lib/api/bucketGetVersioning.js [hunk 1]
```diff
@@ -15,23 +14,25 @@ import services from '../services';
...
+
+    xml.push('<?xml version="1.0" encoding="UTF-8" standalone="yes"?>',
+             '<VersioningConfiguration ' +
+                'xmlns="http://s3.amazonaws.com/doc/2006-03-01/">'
+    );
+
     if (versioningConfiguration && versioningConfiguration.Status) {
-        constructedJSON.VersioningConfiguration.push(
-            { Status: versioningConfiguration.Status });
+        xml.push(`<Status>${versioningConfiguration.Status}</Status>`);
```
Comment (LaurenSpiegel) [inline lib/api/bucketGetVersioning.js]: I assume there is no need to escape this?

File: lib/api/bucketGetVersioning.js [hunk 2]
```diff
@@ -15,23 +14,25 @@ import services from '../services';
...
     }
+
     if (versioningConfiguration && versioningConfiguration.MfaDelete) {
-        constructedJSON.VersioningConfiguration.push(
-            { MfaDelete: versioningConfiguration.MfaDelete });
+        xml.push(`<MfaDelete>${versioningConfiguration.MfaDelete}</MfaDelete>`);
```
Comment (LaurenSpiegel) [inline lib/api/bucketGetVersioning.js]: same

File: lib/api/bucketGetVersioning.js (hunk 1 above)
Comment (bennettbuchanan) [inline lib/api/bucketGetVersioning.js]: Yes, it appears that the only possible values of `versioningConfiguration.Status` and  `versioningConfiguration.MfaDelete` are `undefined`, `'Enabled'`, and `'Suspended'`.
See: https://github.com/scality/S3/blob/master/lib/metadata/BucketInfo.js#L63

Thus, no need for escaping.

**Comment by bennettbuchanan** [comment]
@ironman-machine

**Pull request merged**